Microfinance.sol

```diff
@@ -1,4 +1,63 @@
-pragma solidity ^0.5.16;
+pragma solidity ^0.5.9;
+import "github.com/OpenZeppelin/zeppelin-solidity/contracts/math/SafeMath.sol";
 
-contract Microfinance {
+contract MicroFinance {
+    
+    using SafeMath for uint;
+     
+    struct Loaner{
+        uint loan;
+    }
+    struct Lenders{
+        address Loaner;
+    }
+    // mapping(key_type => value_type)
+    mapping(address => Loaner) loaners;
+    mapping(address => Lenders) lenders;
+    address[] public loanerAccts;
+    address[] public lenderAccts;
+
+    function setLoaner(address _address, uint _loan) public {
+        require(_loan <= 2 ether, "Max 2 ethers");
```

Comment:
'ether' is uint??

Reply:
ether is a value is smartcontract and the require loan to equal less than 2 ether works but when your using it its in wei so 18 zeros

Reply:
I understand!